viewing_party/party.py

```diff
@@ -1,23 +1,122 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if not title or not genre or not rating:

```

Comment:
Excellent use of the guard clause pattern! You could even wait to initialize `new_movie` below the guard clause since if the guard clause returns early, the empty dictionary isn't used.